apps/web/app/api/user/stats/emails/sw/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from "next/server";
```

Comment:
This api route helps in returning an empty message response iff service working is not installed on the browser or fails to intercept a fetch request.